lib/utils/uid.js

```diff
@@ -4,8 +4,7 @@ let counter = 0
  * @link https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L16143
  * @param {string} prefix
  */
-export default function uid(prefix = "") {
+export default function uid(prefix = "bv_") {
     const id = ++counter
```

Comment:
With this be OK with SSR stuff? where the counter never gets reset as long as the server i running?  And will it conflict with any IDs generated by the client (which will start off at 0)?